src/Microsoft.VisualStudio.Web.BrowserLink/ScriptInjectionFilterStream.cs

```diff
@@ -271,10 +271,11 @@ private void DetermineIfFilterShouldBePassthrough(byte[] buffer, int offset, int
             string contentTypeFromHeader = _context.ResponseContentType;
             string path = _context.RequestPath;
 
-            if (!ContentTypeUtil.IsHtml(contentTypeFromHeader))
+            if (!ContentTypeUtil.IsXhtml(contentTypeFromHeader) && !ContentTypeUtil.IsHtml(contentTypeFromHeader))
             {
                 CloseInjectScriptSocketAndBecomePassthrough();
             }
+
             else if (!ContentTypeUtil.IsHtml(path, buffer, offset, count))
```

Comment:
Xhtml and html share some same content and will be viewed as txt/html based on buffer